plugins/node.d/squeezebox_

```diff
@@ -192,13 +192,14 @@ elif [ "$CMD" = "signalstrength" -o "$CMD" = "mixer volume" ]; then
             MAC=$(printf "%b" "player id $ID ?\nexit\n" | $NC $HOST $PORT | cut -d " " -f 4 | sed 's/%3A/:/g')
             VAL=$(printf "%b" "$MAC $CMD ?\nexit\n"| $NC $HOST $PORT | cut -d " " -f 2- | sed "s/$CMD //")
             MAC2=$(echo $MAC| sed 's/://g')
+            [ $VAL -eq 0 ] && VAL=100
```

Comment:
This will set VAL to 100 for every VAL that has value 0.
Is it possible to recognize the wired devices which you want to address with this change?